### PR TITLE
Update introduction.

### PR DESCRIPTION
**Pull request by fmassot**

### CODE REVIEW

File: docs/introduction.md [hunk 1]
```diff
@@ -1,29 +1,51 @@
...
+title: Introduction
 slug: /
 sidebar_position: 1
 ---
 
-Quickwit is a distributed search engine built from the ground up to offer cost-efficiency and high reliability. By mere mortals for mere mortals, Quickwit's architecture is as simple as possible[^1].
+Quickwit is a distributed search engine designed from the ground up to offer cost-efficiency and high reliability on large data sets.
 
-Quickwit is written in Rust and built on top of the mighty [tantivy](https://github.com/tantivy-search/tantivy) library. We designed it to index large datasets.
+Quickwit is particularly well-suited for dealing with large, immutable datasets and a low average QPS$^1$. Its benefits are most apparent in a multi-tenancy or multi-index setting.
```
Comment (guilload) [inline docs/introduction.md:9]:
```suggestion
Quickwit is particularly well-suited for dealing with large, immutable datasets and relatively low average QPS$^1$. Its benefits are most apparent in multi-tenancy or multi-index settings.
```

File: docs/introduction.md [hunk 2]
```diff
@@ -1,29 +1,51 @@
...
 
-## Why Quickwit?
+Common use cases for Quickwit include:
 
-Quickwit is born from the idea that today's search engines are hard to manage and uneconomical when dealing with large datasets and a low QPS[^2] rate. Its benefits are most apparent in a multitenancy or a multi-index setting.
+- Searching through logs, from a small amount to TB of data.
```
Comment (guilload) [inline docs/introduction.md:13]:
```suggestion
- Searching through logs, from small amounts of data to terabytes.
```

File: docs/introduction.md [hunk 3]
```diff
@@ -1,29 +1,51 @@
...
+- Adding full-text search capabilities to [OLAP databases such as ClickHouse](link to blog post).
+- Searching through backups sitting on your cloud storage by adding Quickwit index files on your same storage.
```
Comment (guilload) [inline docs/introduction.md:15]: This needs to be reworded but I have a hard time coming up with something compelling.

File: docs/introduction.md [hunk 4]
```diff
@@ -1,29 +1,51 @@
...
 
-Quickwit allows true decoupled compute and storage.
-We designed it to search straight from object storage like Amazon S3 in a stateless manner.
+# Key features of Quickwit
 
-Imagine hosting an arbitrary amount of indexes on Amazon S3 for $25 per TB/month and querying them with the same pool of search servers and with a subsecond latency.
+Quickwit allows true decoupled compute and storage and we designed it to search straight from deep storage (cloud storage, HDFS, local file system, ...) in a stateless manner. Here is a non-exhaustive list of Quickwit’s key features:
```
Comment (guilload) [inline docs/introduction.md:19]:
```suggestion
Quickwit is designed to search straight from object storage allowing true decoupled compute and storage. Here is a non-exhaustive list of Quickwit’s key features:
```

File: docs/introduction.md [hunk 5]
```diff
@@ -1,29 +1,51 @@
...
 
-Not only is Quickwit more cost-efficient, but search clusters are also easier to operate. One can add or remove search instances in seconds. You can also effortlessly index a massive amount of historical data using your favorite batch technology. Last but not least, Multi-tenant search is now cheap and painless.
+- **Scalable distributed search:** Host an arbitrary amount of indices on S3 and query them with a pool of search servers and with sub-second latency.
```
Comment (guilload) [inline docs/introduction.md:21]:
```suggestion
- **Scalable distributed search:** Host an arbitrary number of indexes on Amazon S3 and answer search queries in less than a second with a small pool of stateless search instances.
```

File: docs/introduction.md [hunk 6]
```diff
@@ -1,29 +1,51 @@
...
+- **Stream indexing:** Plug in your distributed event streaming platform and ingest TB of data. As of today, Quickwit supports Kafka natively.
```
Comment (guilload) [inline docs/introduction.md:22]:
```suggestion
- **Stream indexing:** Ingest TB of data from your favorite distributed event streaming service. As of today, Quickwit supports Apache Kafka natively. The next releases will bring support for more platforms.
```

File: docs/introduction.md [hunk 7]
```diff
@@ -1,29 +1,51 @@
...
+- **Fault-tolerant architecture that won't lose data:** Quickwit ensures **exactly-once** semantics for indexing and your data can be safely stored on highly reliable deep storage like S3.
```
Comment (guilload) [inline docs/introduction.md:23]:
```suggestion
- **Fault-tolerant architecture that won't lose data:** Quickwit achieves **exactly-once** processing for indexing and safely stores your data on highly reliable object storage services such as Amazon S3.
```

File: docs/introduction.md [hunk 8]
```diff
@@ -1,29 +1,51 @@
...
+- **Cloud-native, easy to operate:** Thanks to true decoupled compute and storage, search instances are stateless, you can add or remove search instances in seconds.
```
Comment (guilload) [inline docs/introduction.md:24]:
```suggestion
- **Cloud-native, easy to operate:** Thanks to true decoupled compute and storage, search instances are stateless, add or remove search nodes within seconds.
```

**Review by guilload — COMMENTED** [review]
Left some suggestions.

File: docs/introduction.md (hunk 1 above)
Comment (fulmicoton) [reply docs/introduction.md:9]: The ^1 thing is broken.

File: docs/introduction.md (hunk 3 above)
Comment (fulmicoton) [reply docs/introduction.md:15]: Broken link

File: docs/introduction.md [hunk 9]
```diff
@@ -1,29 +1,51 @@
...
-- [Get started](getting-started/quickstart.md)
+# When to use Quickwit
 
+Quickwit should be a good match if your use case has some of the following characteristics:
 
----
-[^1] ... But not one bit simpler.
+- Your documents are immutable.
+- You are targeting query latencies of 100ms to a few seconds.
+- You have a low average QPS$^1$, typically < 10 QPS on average over the month. This is the case for most search use cases as long as search is not public: enterprise search, log search, email search, security search, ...
```
Comment (fulmicoton) [inline docs/introduction.md:35]: broken footnote link (see above.)

File: docs/introduction.md (hunk 3 above)
Comment (fmassot) [reply docs/introduction.md:15]: Link fixed.

**Pull request merged**